src/Maestro/SubscriptionActorService/PullRequestActor.cs

```diff
@@ -872,8 +872,18 @@ await AddDependencyFlowEventsAsync(
                 MergePolicyCheckResult.PendingPolicies,
                 pr.Url);
 
+            var targetBranchDeps = (await darcRemote.GetDependenciesAsync(targetRepository, targetBranch)).ToList();
```

Comment:
Can you put a comment in there why we do this here?

Reply:
Or this whole block (get original dependencies + replace the froms) could be a named method?

Reply:
Sure, the named method idea sounds nice, will change